src/config/ConfigService.js

```diff
@@ -42,6 +42,9 @@ export default class ConfigService extends Service {
 	initialize() {
 		let config_url = this.App.Config.get('CONFIG_PATH');
 
+		const xConfig = document.getElementsByName('x-config')[0];
```

Comment:
`X-Config`

Reply:
Dashes are not allowed in naming in javascript (parser assume that it is subtract operator). I can use underscore instead.

Reply:
I meant:
```
const xConfig = document.getElementsByName('X-Config')[0];
```